src/EditorFeatures/CSharpTest/Classification/SyntacticTaggerTests.cs

```diff
@@ -102,7 +102,7 @@ public async Task TestTagsChangedAfterDelete()
 
             var checkpoint = new Checkpoint();
 
-            var typeMap = workspace.ExportProvider.GetExportedValue<SyntacticClassificationTypeMap>();
+            var typeMap = workspace.ExportProvider.GetExportedValue<ClassificationTypeMap>();
```

Comment:
got rid of the two type maps.  we only have one type map we use uniformly in roslyn now.